src/ontology/cl-edit.owl

```diff
@@ -19054,13 +19054,20 @@ SubClassOf(obo:CL_0002451 obo:CL_0000048)
 
 # Class: obo:CL_0002453 (oligodendrocyte precursor cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:20142420"^^xsd:string) obo:IAO_0000115 obo:CL_0002453 "The cell type from which oligodendrocytes develop. This cell originates from multiple structures within the developing brain including the medial ganglion eminence and the lateral ganglionic eminence. These cells migrate throughout the central nervous system and persist into adulthood where they play an important role in remyelination of injured neurons."^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:10704434"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:11756508"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:20142420"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:2182078"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:24133281"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:9826671"^^xsd:string) obo:IAO_0000115 obo:CL_0002453 "A progenitor cell of the central nervous system that can differentiate into oligodendrocytes or type-2 astrocytes. This cell originates from multiple structures within the developing brain including the medial ganglion eminence and the lateral ganglionic eminence. These cells migrate throughout the central nervous system and persist into adulthood where they play an important role in remyelination of injured neurons."^^xsd:string)
 AnnotationAssertion(oboInOwl:created_by obo:CL_0002453 "tmeehan"^^xsd:string)
 AnnotationAssertion(oboInOwl:creation_date obo:CL_0002453 "2010-11-10T02:51:34Z"^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:24133281"^^xsd:string) oboInOwl:hasBroadSynonym obo:CL_0002453 "O-2A/OPC"^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:11756508"^^xsd:string) oboInOwl:hasBroadSynonym obo:CL_0002453 "O2A/OPC"^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:2253328"^^xsd:string) oboInOwl:hasBroadSynonym obo:CL_0002453 "oligodendrocyte-type 2 astrocyte (O-2A) progenitor cell"^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:11756508"^^xsd:string) oboInOwl:hasBroadSynonym obo:CL_0002453 "oligodendrocyte-type-2 astrocyte progenitor cell"^^xsd:string)
```

Comment:
this looks like a duplicate from the above except without (O-2A) - in which case i don't think there's a need. search stuff would be able to pick it up without having this synonym